### ISSUE TITLE
can not exclude files

### ISSUE THREAD

**Issue by zap-mueda** [brigade/scss-lint#740]
hello,
I am setting `.scss-lint.yml` like this.

```yml
scss_files: 'src/scss/**/*.scss'

exclude: 'src/scss/file/**'
```

I want to exclude some files, but not ignore... 
could you give me advice?

**Comment by sds** [comment]
Which version of `scss-lint` are you running, @zap-mueda?

Please ensure you're installing `scss_lint` (with an underscore) and not `scss-lint`.

**Comment by zap-mueda** [comment]
version of `scss-lint` is v0.47.0, and i'm installing `scss_lint` (with an underscore).
thanks @sds

**Comment by sds** [comment]
I can't reproduce your problem, @zap-mueda. I created the following directories:

```
.
└── .scss-lint.yml
└── src
    └── scss
        ├── error.scss
        └── file
            └── excluded.scss
```

Using the configuration you specified above, it correctly reports lints in `error.scss` but not in `excluded.scss`.

Make sure you are placing your `.scss-lint.yml` file in the correct location. Hope this helps.

**Comment by zap-mueda** [comment]
thanks @sds
sorry, I using the text editor only, didn't try the command line.
I did well!
